packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/Contracts.jsx

```diff
@@ -18,7 +18,7 @@ import NavigationList, {
 } from 'cozy-ui/transpiled/react/NavigationList'
 
 import { getAccountLabel } from './bankAccountHelpers'
-import EditContractModal from './EditContract'
+import EditContract from './EditContract'
```

Comment:
It's not a modal anymore? Wondering if we shouldn't have done the opposite: renaming the filename. Having Modal in the filename tells us that we need to pass props like onDismiss and so on